Description: String typo fixes
 Fix various language nits in the program strings.
Author: Malcolm Parsons
Bug-Ubuntu: https://bugs.launchpad.net/bugs/62854

=== modified file 'pbbcmd/pbbcmd.c'
Index: pbbuttonsd-0.8.1/pbbuttonsd.1
===================================================================
--- pbbuttonsd-0.8.1.orig/pbbuttonsd.1
+++ pbbuttonsd-0.8.1/pbbuttonsd.1
@@ -159,7 +159,7 @@ VolumeDownKey    \-\-       Decrease vol
 
 MuteKey          \-\-       Disable/enable sound output. The volume level
                           won't change. Pressing any other volume con\-
-                          trol key will automaticaly enable sound out\-
+                          trol key will automatically enable sound out\-
                           put again.
 
 EjectCDKey       \-\-       Ejects a CDROM from CDROM drive. If necessary
@@ -464,12 +464,12 @@ a bit the first time. But after roundabo
 cycles the predicted time and the battery warnings will be rather stable
 and reliable.
 .PP
-The database is updated continously so that any aging of the battery is taken
+The database is updated continuously so that any aging of the battery is taken
 into account. When \fBpbbuttons\fP tells you now, you still have 10 minutes
 left on battery, then you \fBwill\fP have 10 minutes left even if your
 battery is growing old and the capacity decreases.
 .SH IBaM and multiple batteries
-Currently IBaM is not able to handle multiple batteries seperatly. It always
+Currently IBaM is not able to handle multiple batteries separately. It always
 sums up all batteries in system and calculates with the result. Therefore the
 IBaM statistics will be influenced if you change your battery configuration
 (adding or removing batteries).
Index: pbbuttonsd-0.8.1/pbbuttonsd.cnf.5
===================================================================
--- pbbuttonsd-0.8.1.orig/pbbuttonsd.cnf.5
+++ pbbuttonsd-0.8.1/pbbuttonsd.cnf.5
@@ -579,7 +579,7 @@ also set differently dependent on the po
 .in +7
 This options could be used to limit the working range of the automatic
 brightness controller in general or depending on the power source, which
-is very usefull most of the time. It might help to save energy when running
+is very useful most of the time. It might help to save energy when running
 on battery.
 .TP
 \fBLCD_IllumUpKey\fR = \fIkey\fR (default: KEY_BRIGHTNESSUP)
@@ -689,7 +689,7 @@ backlight will be switched off on blank.
 .in +7
 Recent kernel versions will do this automatically because the backlight
 controller is part of the framebuffer or graphics card driver. If you
-relialize any problems with the backlight, like the diplay remains dark
+relialize any problems with the backlight, like the display remains dark
 after wakeup or cover open, set this option to \'no\'.
 .TP
 \fBDimFullyDark\fR = \fI[yes | no]\fR (default: no)
@@ -882,7 +882,7 @@ kernels might freeze once and again. The
 please use this option with care.
 .PP
 .in +7
-This option is only usefull if \fBmouseemu\fR is not running. \fBMouseemu\fR
+This option is only useful if \fBmouseemu\fR is not running. \fBMouseemu\fR
 offer this and some more trackpad related functions. Set this option
 to \'no\', if you have \fBmouseemu\fR running.
 .TP
